cthreads.c

```diff
@@ -136,7 +136,11 @@ void cthreads_thread_exit(void *code) {
   #endif
 
   #ifdef _WIN32
-    #if defined  __WATCOMC__ || _MSC_VER || __DMC__
+    /* NOTE: This gives warnings on MSVC, so I removed this
+     * for the oldest version of it I could test (19.20). */
+    /* TODO: Test with the other described compilers if this
+     *       is _really_ necessary. */
```

Comment:
```suggestion
     *       is REALLY necessary. */
```

Not sure, just a suggestion, for me both work.